pkg/storaged/anaconda.jsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import client from "./client.js";
+
+export const AnacondaAdvice = () => {
+    if (!client.in_anaconda_mode())
+        return null;
+
+    // Nothing yet.
+    return null;
```

Comment:
I'm fine with that for the first "secret" version, but what's the idea what this should eventually be? Some kind of inline help for what admins are supposed to do on the page, and how to go back?

Reply:
I leave that up to you, just wanted to point it out.